crates/api_models/src/payments.rs

```diff
@@ -576,6 +576,7 @@ pub enum WalletData {
 }
 
 #[derive(Eq, PartialEq, Clone, Debug, serde::Deserialize, serde::Serialize, ToSchema)]
+#[serde(rename_all(serialize = "camelCase", deserialize = "snake_case"))]
```

Comment:
No google provides the token details in camelCase always and in our system we are consuming in it snake_case because other payment_methods also follow snake_case. so when accepting inputs it should be snake_case and when consuming it should camelCase